src/utils/add.trust.ts

```diff
@@ -53,6 +54,7 @@ const getPageProperties = (
       subtitle: ADD_TRUST_TEXTS.subtitle,
     },
     isUpdate,
+    isFeatureFlagCeaseTrustsEnabled: isActiveFeature(config.FEATURE_FLAG_ENABLE_CEASE_TRUSTS),
```

Comment:
Do we need this? The template could in theory get hold of the feature flag value by doing something like this:

`{% if FEATURE_FLAG_ENABLE_CEASE_TRUSTS %}`

And then set a local template variable, if required.